src/Input/Input.js

```diff
@@ -176,6 +182,15 @@ export default class Input extends Component {
     }
   }
 
+  shouldComponentUpdate(nextProps, nextState, nextContext) {
```

Comment:
I'll look at this closer again. I put it in there temporarily to help track down an issue I was having with the `FormControl` causing an infinite loop when an `onFocus` handler was passed to `SelectField`.